textacy/datasets/wikipedia.py

```diff
@@ -82,7 +88,7 @@
 re_brackets = re.compile(r'\[\s?\]|\(\s?\)')
 
 re_comments = re.compile('<!--.*?-->', flags=re.UNICODE | re.DOTALL)
-re_categories = re.compile(r'\[\[Category:[^\]\[]*\]\]', flags=re.UNICODE)
+re_categories = re.compile(r'\[\[(%s)[^\]\[]*\]\]' % '|'.join(MAPPING_CAT.values()), flags=re.UNICODE)
```

Comment:
nit: Could you use `str.format()` instead of the older `%s` form?

Reply:
sure